plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -129,10 +132,20 @@ const BtnsWrapper = styled.div<{ isMobile?: boolean }>`
   width: ${({ isMobile }) => (isMobile ? "100%" : "318px")};
 `;
 
-const ImgWrapper = styled.div`
+const ImgWrapper = styled.a<{ imgUrl: string }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: 305px;
   height: 159px;
   cursor: pointer;
+  background: ${props =>
+    `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${props.imgUrl})`};
+
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)) url(welcomeScreenVideo);
```

Comment:
Is the second `background` working? 
It seems the `linear-gradient` uses the same color for gradient, is it for some reason?

Reply:
I updated some on this style. Please help check.